bundles/org.openhab.binding.hue/src/main/resources/OH-INF/i18n/hue.properties

```diff
@@ -182,3 +273,14 @@ actionDesc = Send a light command with a custom fade time.
 # discovery results
 
 discovery.group.all_lights.label = All lights
+
+# api v2 dynamic actions
+
+dynamics.action.label = Send a dynamic command.
+dynamics.action.description = Send a dynamic command to a device, room or zone.
+dynamics.channel.label = Target Channel
+dynamics.channel-id.description = The channel ID of the channel to send the command to.
+dynamics.command-id.label = Target Command
```

Comment:
Take care: dynamics.command.label is used in the java file.

Reply:
Good catch. Done